process/sync/baseSync.go

```diff
@@ -121,6 +123,15 @@ func (boot *baseBootstrap) loadBlocks(
 		}
 	}
 
+	firstNonceToDeleteFromStorer := highestNonceInStorer - lastBlocksToSkip + 1
+
+	for i := firstNonceToDeleteFromStorer; i <= highestNonceInStorer; i++ {
```

Comment:
firstNonceToDeleteFromStorer is actually the i from the previous for

Reply:
Refactored